### PR TITLE
10.9 Release Notes

### PR DESCRIPTION
**Pull request by pmaier1**
Draft mode. Still some parts missing.

### CODE REVIEW

File: modules/ROOT/pages/server_release_notes.adoc
Comment (phil-davis) [inline modules/ROOT/pages/server_release_notes.adoc:85]:
```suggestion
In certain scenarios it is not desired that all users of an ownCloud installation can create public links and share data publicly, respectively. To cover for this, ownCloud Server 10.9 comes with a new configuration option for administrators that allows to restrict the creation of public links to users in certain groups (https://github.com/owncloud/core/pull/38980[#38980]).
```
```suggestion
In certain scenarios it is not desired that all users of an ownCloud installation can create public links and share data publicly, respectively. To cover for this, ownCloud Server 10.9 comes with a new configuration option for administrators that allows restricting the creation of public links to users in certain groups (https://github.com/owncloud/core/pull/38980[#38980]).
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (phil-davis) [inline modules/ROOT/pages/server_release_notes.adoc:91]:
```suggestion
In some cases it is necessary to manage storage locations of users, e.g., if you want to distribute them across multiple storages. To facilitate this process, Server 10.9 introduces a new _occ_ command (`occ user:move-home`) that allows to move user home folders to a new location (https://github.com/owncloud/core/pull/39136[#39136]). The command takes care of moving the physical data and takes care of updating all other related information.
```
```suggestion
In some cases it is necessary to manage storage locations of users, e.g., if you want to distribute them across multiple storages. To facilitate this process, Server 10.9 introduces a new _occ_ command (`occ user:move-home`) that allows moving user home folders to a new location (https://github.com/owncloud/core/pull/39136[#39136]). The command takes care of moving the physical data and takes care of updating all other related information.
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (phil-davis) [inline modules/ROOT/pages/server_release_notes.adoc:105]: This is a bit tricky to describe - subadmins cannot add just any user to a group that they administer.
If the subadmin administers groups groupA, groupB and groupC, if Brian is already a member of groupA then the subadmin can also make Brian a member of groupB and groupC.

I guess people can read that in the linked PR. I am having trouble thinking of a short edit to this line that will be useful.

Comment (pmaier1) [reply modules/ROOT/pages/server_release_notes.adoc]: Yeah, only those users the group admin "knows of" can be added, true. I tried to make it a bit more precise.

File: modules/ROOT/pages/server_release_notes.adoc
Comment (jnweiger) [inline modules/ROOT/pages/server_release_notes.adoc:100]: * The following app updates are included in 10.9.0beta1:
   - files_mediaviewer : v1.0.5
   - files_pdfviewer : v1.0.1
   - files_texteditor : v2.4.1
   - guests : v0.9.3
   - oauth2 : v0.5.2
   - openidconnect : v2.1.0
   - web : v4.5.0
   - windows_network_drive : v2.1.0
   - wopi : v1.5.1

**Review by jnweiger — DISMISSED** [review]
commented all the app updates that I saw while building beta1

**Comment by mmattel** [comment]
@pmaier1 you may want to add the MariaDB 10.6 issue https://github.com/owncloud/docs/issues/4115
because there is no single but a bunch of PR's adressing this topic which all reference to the issue, still working on it, the list of PR's increases 😅 ...
(or you add all those single PR's to the list instead the issue - will tell when the last one is made)

**Comment by pmaier1** [comment]
> @pmaier1 you may want to add the MariaDB 10.6 issue #4115

Thanks but I'd prefer to add a link to the doc page once the PRs are merged.

**Comment by mmattel** [comment]
> Thanks but I'd prefer to add a link to the doc page once the PRs are merged.

Done, for the reference, this is the PR https://github.com/owncloud/docs/pull/4360 (Upgrade guide for MariaDB 10.6)
The PR also backported to the prepared doc 10.9 branch which is currently not available on the web.
This is the doc link available on the web: https://doc.owncloud.com/server/next/admin_manual/maintenance/upgrading/database_upgrade.html

**Comment by jnweiger** [comment]
We have two more apps updates from beta1 to RC1:
* web 4.6.0
* user_ldap 0.16.0-rc2
*

File: modules/ROOT/pages/server_release_notes.adoc
Comment (mmattel) [inline modules/ROOT/pages/server_release_notes.adoc:71]: FYI, the feature is documented in (admin and user manual):
https://doc.owncloud.com/server/10.9/admin_manual/configuration/files/manual_file_locking.html
https://doc.owncloud.com/server/10.9/user_manual/files/manual_file_locking.html

File: modules/ROOT/pages/server_release_notes.adoc
Comment (mmattel) [inline modules/ROOT/pages/server_release_notes.adoc:90]: FYI, the feature is documented at:
https://doc.owncloud.com/server/10.9/admin_manual/configuration/user/user_management.html
https://doc.owncloud.com/server/10.9/admin_manual/configuration/server/occ_command.html#user-commands

Note the related new `user:home` commands which are described at the same location

**Comment by mmattel** [comment]
**Some change log notes**

I dont know if important, but here is a list I have found what has been added in docs maybe additional relevant for a change log. I am not sure about the encryption part if it is core relevant or encryption only:

* https://github.com/owncloud/docs/pull/4441/files (add more known and patched jquery issues)
* https://github.com/owncloud/docs/pull/4404/files (New occ Activity commands)
* https://github.com/owncloud/docs/pull/4403 (New: email configuration: configurable recipient address)
* https://github.com/owncloud/docs/pull/4372 (Cleanup of occ dav commands) (--local option)
* https://github.com/owncloud/docs/pull/4299 (Add new encryption:fix-encrypted-version option -i)
* https://github.com/owncloud/docs/pull/4298 (Add new app:list option -m)
* https://github.com/owncloud/docs/pull/4243 (Add new user:list option) (show-all-attributes)
* https://github.com/owncloud/docs/pull/4255 (Adding occ command setting to hide the upload estimation)
* https://github.com/owncloud/docs/pull/4152 (Preview webp)
* https://github.com/owncloud/docs/pull/4148 (Jpeg quality)

**Merge notes**
* Consider to merge this PR **after** setting the 10.9 tag in core. This is because if you use `latest` in the URL, it will only point to 10.9 if the tag has been set correctly. See that I have used in the comments above `10.9` instead of `latest` to give you direct access to the content, but it should be `latest` when used.
* I approve this now because the current content is ok to me and you need the have the freedom to merge when you think it is ready.

**Review by mmattel — APPROVED** [review]

**Comment by pmaier1** [comment]
Thanks! The relevant points of the list are already part of the release notes. This is ready for final review, I think.

File: modules/ROOT/pages/server_release_notes.adoc
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:109]:
```suggestion
* The handling of inaccessible federated shares has been improved to avoid removing temporarily unavailable shares. https://github.com/owncloud/core/pull/38474[#38474]
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:111]:
```suggestion
* Issues with updating apps via the Market app occurring under certain circumstances have been resolved. https://github.com/owncloud/core/pull/39108[#39108]
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:115]:
```suggestion
* The "Home" icon in the breadcrumb of the Files app has been replaced with the icon and text of the respective view (e.g., All Files, Deleted Files) so that its functionality is clearer to the user. https://github.com/owncloud/core/pull/39367[#39367]
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:117]:
```suggestion
* The "Open with" menu which is shown when a user can choose between multiple applications/actions for a certain file type (e.g., ONLYOFFICE, Collabora Online, PDF Viewer) has been improved to only show relevant actions. If there's only one relevant action available, clicking on the file name will directly conduct this action. https://github.com/owncloud/core/pull/39541[#39358]
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:118]:
```suggestion
* Support for Collabora Online Draw has been added. https://github.com/owncloud/core/pull/39594[#39594]
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:120]:
```suggestion
* The quality of preview images (e.g., in the Media Viewer) can now be configured. https://github.com/owncloud/core/blob/master/config/config.sample.php#L996[in _config.php_]. https://github.com/owncloud/core/pull/39349[#39349]
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:127]:
```suggestion
* The group listing of the user management in the web interface has been improved with regard to usability. https://github.com/owncloud/core/pull/39262[#39262]
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:143]:
```suggestion
All xref:server_release_notes.adoc#known-issues[known issues in Server 10.8] have been fixed.
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:147]:
```suggestion
=== Known Issues
```

**Review by EParzefall — APPROVED** [review]
Some language improvements necessary

File: modules/ROOT/pages/server_release_notes.adoc
Comment (mmattel) [inline modules/ROOT/pages/server_release_notes.adoc:71]:
```suggestion
Version 10.9 introduces a new subfeature that allows administrators to define groups of users that are entitled to unlock locked files, independent of whether they have set the lock initially or not (https://github.com/owncloud/core/pull/38222[#38222]). Users in such groups can regain control over files, e.g., if a lock owner is unavailable and a long timeout for file locks is used. Administrators can configure this in the _Manual File Locking_ section of the _General_ admin settings. More information, refer to the https://doc.owncloud.com/server/latest/admin_manual/configuration/files/manual_file_locking.html[ownCloud documentation].
```

File: modules/ROOT/pages/server_release_notes.adoc
Comment (mmattel) [inline modules/ROOT/pages/server_release_notes.adoc:90]:
```suggestion
For more information, refer to the https://doc.owncloud.com/server/latest/admin_manual/configuration/server/occ_command.html#user-commands[ownCloud documentation about occ] and https://doc.owncloud.com/server/latest/admin_manual/configuration/user/user_management.html#moving-the-user-home[about user management].
```

**Comment by mmattel** [comment]
Squashed all commits and removed a stale review.

Propose to merge right **AFTER** the tag in core has been set to 10.9

@pmaier1 you may want to press the merge button when setting the tag has been done 😃

File: modules/ROOT/pages/server_release_notes.adoc
Comment (phil-davis) [inline modules/ROOT/pages/server_release_notes.adoc:89]: `occ user:home` by itself is not a working command. The two commands are:
`occ user:home:list-dirs`
`occ user:home:list-users`

Maybe some words here should be changed, or is the current text enough for summary release notes?

Comment (mmattel) [inline modules/ROOT/pages/server_release_notes.adoc:89]:
```suggestion
In some cases it is necessary to manage storage locations of users, e.g., if you want to distribute them across multiple storages. To facilitate this process, Server 10.9 introduces a new _occ_ command (`occ user:move-home`) that allows moving user home folders to a new location (https://github.com/owncloud/core/pull/39136[#39136]). The command takes care of moving the physical data and of updating all other related information. In addition, new commands like the `occ user:home:list-dirs` which lists the available storage locations of users and `occ user:home:list-users` which lists users belonging to a storage location have been added.
```

**Review by phil-davis — APPROVED** [review]
LGTM

**Comment by phil-davis** [comment]
Issue #4523 - should we add a "PHP 7.3 Deprecation Note" to the release notes?

**Pull request merged**
